libs/core/src/main/java/com/sourcefuse/jarc/core/constants/Constants.java

```diff
@@ -0,0 +1,25 @@
+package com.sourcefuse.jarc.core.constants;
+
+public class Constants {
```

Comment:
Just keep enum in this file. No need to have constants class.

Reply:
Done